pkg/api/handlers/compat/containers.go

```diff
@@ -126,6 +126,12 @@ func ListContainers(w http.ResponseWriter, r *http.Request) {
 	// the all setting and always deal with all containers.
 	if len((*filterMap)["status"]) > 0 {
 		all = true
+		stat, err := filters.GenerateContainerFilterFuncs("status", (*filterMap)["status"], runtime) // pass actual value that the user wants
+		if err != nil {
+			utils.InternalServerError(w, err)
+			return
+		}
+		filterFuncs = append(filterFuncs, stat)
```

Comment:
I don't think this is necessary, the filter functions should have already been generated above.

Reply:
correct me if I am wrong, but I interpreted this part of the code as only passing status to the generateFilters function is all is also true which is not the expected behavior? or is this taken care of further up in the code?

Reply:
When filters are set they are already parsed before, see line 114-123. So we should not parse them again.